src/Repository/AdherentRepository.php

```diff
@@ -549,4 +549,25 @@ public function countCommitteeSupervisorsByGenderForReferentManagedArea(Adherent
             ->getArrayResult()
         ;
     }
+
+    public function countCommitteeMembersByGenderForReferentManagedArea(Adherent $referent): array
+    {
+        if (!$referent->isReferent()) {
+            throw new \InvalidArgumentException('Adherent must be a referent.');
+        }
+
+        return $this->createQueryBuilder('a', 'a.gender')
+            ->select('a.gender, COUNT(DISTINCT a) AS count')
+            ->join('a.memberships', 'cm')
+            ->join(Committee::class, 'c', 'WITH', 'c.uuid = cm.committeeUuid')
+            ->innerJoin('c.referentTags', 'tag')
```

Comment:
`join` is an alias of `innerJoin` method.
can you use one or the other but not both, please?